src/dev/dev-server.ts

```diff
@@ -37,10 +37,22 @@ export const startDevServer = async (options: StartDevServerOptions) => {
 
   const stderr = options.stderr ?? process.stderr
 
+  const buildEvents = new EventEmitter()
```

Comment:
can we [type this](https://www.npmjs.com/package/typed-emitter)? we might want to tap into this in the future, for integrations and stuff

Reply:
also nice to know what kinds of events are possible up front for code visibility/docs purposes